City-of-Helsinki/drupal-helfi-etusivu
# Platta 4664 and uhf 8204

# [UHF-8204](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8204)
PLATTA-4664 and UHF-8204
With changes from https://github.com/City-of-Helsinki/drupal-module-helfi-azure-fs/pull/10
the image derivatives should not be created during page request time

## What was done
Removed image style generation on token replacement for og_image

## How to install
* Check if this PR was merged and update the helfi_azure_fs to latest
* Make sure your instance is up and running on correct branch.
* `git checkout PLATTA-4664-and-UHF-8204`
* Run `make drush-cr`

## How to test
* Your local environment should use azure blob storage (see settings.php for needed env vars)
* Upload a new image on a news item and check the og_image tag in head in the detail page is referencing the sites default files.
* Open the og_image link in new tab, the image should be visible
* Clear the cache and check again the og_image  link in header. now it should point to blobstorage link, instead of local drupal.

## Designers review


* [ X] This PR does not need designers review

## Other PRs
* Link to other PR](https://github.com/City-of-Helsinki/drupal-module-helfi-azure-fs/pull/10)


[UHF-8204]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8204?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
